files/gridlex/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "Laurent G."
+github = "https://github.com/devlint/gridlex"
+homepage = "http://gridlex.devlint.fr"
+description = "Gridlex is just a CSS Flexbox Grid System "
+mainfile = "dist/gridlex.min.js"
```

Comment:
1. Where min.js came from? You only included min.css
2. Dont use `dist/`. Just `gridlex.min.js` is correct